src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -132,6 +134,8 @@ public void execute(BuildResultAdapter buildResult) {
                 tagIde("IntelliJ IDEA", props.get(SYSTEM_PROP_IDEA_VERSION).get());
             } else if (props.get(SYSTEM_PROP_ECLIPSE_BUILD_ID).isPresent()) {
                 tagIde("Eclipse", props.get(SYSTEM_PROP_ECLIPSE_BUILD_ID).get());
+            } else if (props.get(ENV_VAR_VSCODE_PID).isPresent()) {
+                tagIde("VSCode", "");
```

Comment:
We should tag it as their [brand guidelines](https://code.visualstudio.com/brand#brand-name) ask

```suggestion
                tagIde("VS Code", "");
```

Reply:
It's confusing because it also says:

> **Things we're <ins>not</ins> ok with**
> Using the product name — Visual Studio Code or VS Code — on your website, product, service, or in your domain names.

Reply:
Not advocating for that though. No one is going to use `tag:Visual Studio Code`. Instinctively it will be `tag:VS Code`.